languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34124,77 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+     <rulegroup id="VIRGULE_TEMPS" name="virgule temps">
+         <rule>
+             <pattern>
+                 <token postag="R pers suj.*" postag_regexp="yes"/>
+                 <unify>
+                     <feature id="person"/>
+                     <feature id="tense"/>
+                     <marker>
+                         <token postag="V.*" postag_regexp="yes"/>
+                         <unify-ignore>
+                             <token postag="R pers suj.*" postag_regexp="yes" regexp="yes">(?-i)[a-z].*</token>
+                         </unify-ignore>
+                         <token postag="V.*" postag_regexp="yes"/>
+                     </marker>
+                 </unify>
+             </pattern>
+             <message>Une virgule semble être requise dans ce contexte.</message>
+             <suggestion>\1 \2, \3 \4</suggestion>
+             <example correction="Nous montons, nous descendons"><marker>Nous montons nous descendons</marker> nous ne savons plus où aller.</example>
```

Comment:
Here's why the test fails: compare where the `<marker>` is in your `<pattern>` compared to where it is in your `<example>`…

Reply:
Indeed. I made the necessary changes so it should be working now.

Reply:
It does :)